### PR TITLE
Add TECviva Token 

### PR DESCRIPTION
**Pull request by mzfshark**

### CODE REVIEW

**Comment by ONETheo** [comment]
No issues business side.

**Comment by hedgeme** [comment]
Thanks

**Comment by hedgeme** [comment]
We are still pending merger. Is there anything else needed to progress forward?

File: tokenlist.json [hunk 1]
```diff
@@ -131,6 +131,14 @@
...
       "logoURI": "https://raw.githubusercontent.com/harmony-one/swap-token-list/main/assets/NEURONS/logo.png"
-    }
+    },
+    {
+      "chainId": 1666600000,
+      "address": "0x2DE61623042aD1F5564C6BdCa1D7bC5982CB4b81",
+      "symbol": "TEC",
+      "name": "TEC Viva",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/harmony-one/swap-token-list/main/assets/TECViva/logo.png"
```
Comment (msvstj) [inline tokenlist.json:141]: Folder name should match token symbol. Other than that, no issues.
Contract is verified.

**Review by msvstj — CHANGES_REQUESTED** [review]
Folder name should match token symbol. Other than that, no issues.
Contract is verified.

File: tokenlist.json (hunk 1 above)
Comment (mzfshark) [reply tokenlist.json:141]: done.

**Review by msvstj — APPROVED** [review]

File: tokenlist.json (hunk 1 above)
Comment (msvstj) [inline tokenlist.json:141]: Please change logoURI path in .json file as well.

**Review by msvstj — CHANGES_REQUESTED** [review]

**Review by msvstj — APPROVED** [review]

**Pull request merged**